app/assets/javascripts/controllers/middleware_server/middleware_server_controller.js

```diff
@@ -92,6 +92,37 @@ function MwServerController($scope, miqService) {
     miqService.sparkleOn();
     $scope.$broadcast('mwAddDeploymentEvent', $scope.deployAddModel);
   };
+
+  /////////////////////////////////////////////////////////////////////////
+  // Add JDBC Driver
+  /////////////////////////////////////////////////////////////////////////
+
+  $scope.jdbcDriverModel = {};
+  $scope.jdbcDriverModel.serverId = angular.element('#server_id').val();
```

Comment:
Can't we call the `$scope.resetJdbcDriverForm();` here instead of duplicating the code? It'd emit also the `mwAddJdbcDriverReset`, but it shouldn't be a problem.